awx/ui/client/src/license/license.partial.html

```diff
@@ -122,6 +122,19 @@
 							</label>
 						</div>
 					</div>
+					<div class="form-group License-detailsGroup License-detailsGroup--withSeparator">
+						<div class="checkbox">
+							<label class="License-details--label">
+								<input type="checkbox" ng-model="newLicense.pendo" ng-disabled="!user_is_superuser" required>
+								<translate>By default, Tower collects and transmits analytics data on Tower usage to Red Hat. This data is used to enhance future releases of the Tower Software and help streamline customer experience and success. For more information, see
+										<a target="_blank"
+												href="http://docs.ansible.com/ansible-tower/latest/html/installandreference/user-data.html#index-0">
+												this Tower documentation page
+										</a>.
```

Comment:
There's an extra space b/t "page" and the period here at the end of the sentence. 

Also a very small nitpick, but in the card there's an extra sentence at the end: "Uncheck this box to disable this feature." Without this sentence it's not obvious what action the user is taking by checking/unchecking the box.